README.md

```diff
@@ -333,6 +333,25 @@ pending swaps after a restart.
 Information about pending swaps is stored persistently in the swap database.
 Its location is `~/.loopd/<network>/loop.db`.
 
+## Authentication and transport security
+
+The gRPC and REST connections of `loopd` are encrypted with TLS by default and
+secured with macaroon authentication the same way `lnd` is.
+
+If no custom loop data directory is set then the TLS certificate is stored in
```

Comment:
loop dir (`--loopdir`) or loop data dir (`--datadir`)? It is still a bit confusing which file is sought where.

Reply:
Perhaps the command line help should clarify that `loopdir` and the other path flags are mutually exclusive?